graphics/src/layer.rs

```diff
@@ -323,3 +334,21 @@ unsafe impl bytemuck::Zeroable for Quad {}
 
 #[allow(unsafe_code)]
 unsafe impl bytemuck::Pod for Quad {}
+
+#[derive(Debug)]
+/// A collection of meshes.
+pub struct Meshes<'a>(pub Vec<Mesh<'a>>);
+
+impl<'a> Meshes<'a> {
+    /// Returns the number of total vertices & total indices of all [`Mesh`]es.
+    pub fn attribute_count(&self) -> (usize, usize) {
+        self.0
+            .iter()
+            .map(|Mesh { buffers, .. }| {
+                (buffers.vertices.len(), buffers.indices.len())
+            })
+            .fold((0, 0), |(total_v, total_i), (v, i)| {
+                (total_v + v, total_i + i)
+            })
+    }
+}
```

Comment:
Instead of a newtype without opacity, we could expose this method as a function somewhere else and make it take `&[Mesh<'a>]`.